### ISSUE TITLE
HFI Calculator: SPIKE : Generic admin (station admin)

### ISSUE THREAD

**Issue by Sybrand** [bcgov/wps#1937]
**Describe the task**
Evaluate the possibility of using a generic administration interface for station administration. 

A quick google search brings up two potential options:
- https://accent-starlette.github.io/starlette-admin/ 
- https://fastapi-admin.github.io/ for inspiration

**Acceptance Criteria**
- [ ] Pick the most promising frameworks to give a whirl. (2 or 3 max)
- [ ] Evaluate all the frameworks (potential here for working in parallel!)
- [ ] Get a quick and dirty example working in our dev environment, where you can edit, delete and add weather stations.
- [ ] Have an answer on how authentication will work (ideally we should be able to use key cloak)

**Additional Context**
Try to limit evaluation per framework to 1 day.

**Comment by conbrad** [comment]
fastapi-admin won't work for us, it only works with TortoiseORM not sqlachemy.
I got an admin interface working with: https://github.com/aminalaee/sqladmin
but I had to disable the mounting of the frontend paths due to routing issues.

**Comment by conbrad** [comment]
![Screen Shot 2022-04-25 at 2.53.06 PM.png](https://images.zenhubusercontent.com/60464f96048353febabb5c8d/e6bcdacf-ac10-4674-801c-35367537d326)

**Comment by conbrad** [comment]
Trying starlette admin, unable to install with these instructions: https://accent-starlette.github.io/starlette-admin/#getting-started-installation

Managed to get poetry to resolve the URL, git+https://github.com/accent-starlette/starlette-admin.git@master, but then it fails with:

```
    File "/usr/local/Cellar/python@3.9/3.9.12/Frameworks/Python.framework/Versions/3.9/lib/python3.9/codecs.py", line 322, in decode
      (result, consumed) = self._buffer_decode(data, self.errors, final)
  UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```

**Comment by conbrad** [comment]
Closing in favor of building out the admin interface directly in HFI since this will be a fire centre user task
